test/org/sagebionetworks/bridge/hibernate/HibernateHelperTest.java

```diff
@@ -41,14 +42,16 @@
 
     private HibernateHelper helper;
     private Session mockSession;
-
+    private SessionFactory mockSessionFactory;
+    
     @Before
     public void setup() {
         // mock session
         mockSession = mock(Session.class);
+        mockSessionFactory = mock(SessionFactory.class);
 
         // Spy Hibernate helper. This allows us to mock execute() and test it independently later.
-        helper = spy(new HibernateHelper());
+        helper = spy(new HibernateHelper(mockSessionFactory, TestConstants.PASSTHROUGH_PERSISTENCE_CONVERTER));
```

Comment:
Now we can't tell the difference between the converter not being called vs the converter returning the original exception, nor can we tell the difference between the converter returning the original exception vs the converter converting the exception.

Maybe a simple a mock would be better, so we can control it in our test.